packages/react/src/ReactFetch.js

```diff
@@ -74,7 +74,13 @@ if (enableCache && enableFetchInstrumentation) {
         url = resource;
       } else {
         // Normalize the request.
-        const request = new Request(resource, options);
+        // if resource is not a string or a URL (its an instance of Request)
+        // then do not instantiate a new Request but instead
+        // reuse the request as to not disturb the body in the event it's a ReadableStream.
+        const request =
+          typeof resource === 'string' || resource instanceof URL
```

Comment:
This could in theory be a URL object from a different iframe/realm which is a bit annoying.